torch/_dynamo/external_utils.py

```diff
@@ -61,15 +64,15 @@ def call_hook(
     return result
 
 
-def wrap_numpy(f: Callable[..., Any]) -> Callable[..., Any]:
+def wrap_numpy(f: Callable[_P, pytree.PyTree]) -> Callable[_P, pytree.PyTree]:
```

Comment:
```suggestion
def wrap_numpy(f: Callable[_P, _R]) -> Callable[_P, _R]:
```

Typevar `_R` is the better one than `PyTree`.

Reply:
@XuehaiPan Follow up PR? I think this caused typing issues somewhere with optree's typing system?

Reply:
`pytree.PyTree` is just an alias to `typing.Any`.